website/content/docs/commands/job/tag.mdx

```diff
@@ -0,0 +1,87 @@
+---
+layout: docs
+page_title: 'Commands: job tag'
+description: |
+  Use the job tag commands to manage versions. The job tag apply command saves a job version tag, and the job tag unset command removes a job version tag.
+---
+
+
+# Command: job tag
+
+Use the `job tag` command to manage tags for job versions. `job tag` has the
+following subcommands:
+
+- [`job tag apply`](#apply): Save a job version tag.
+- [`job tag unset`](#unset): Remove a tag from a job version.
+
+## Usage
+
+```shell
+nomad job tag <subcommand> [options] [args]
+```
+
+## General options
+
+Use these optional general options with the subcommands.
+
+@include 'general_options.mdx'
+
+## Apply
+
+Use `job tag apply` to create or modify a version tag.
```

Comment:
I'd consider adding two things here:
- A brief explanation about what the tag does
- A note that tag names are unique per job

Something like "Applying a tag to a version prevents it from being garbage-collected, and makes it accessible by name when comparing versions"

Reply:
done - added a partial and included on cli/api